src/recorder.ts

```diff
@@ -183,6 +190,29 @@ export default async (
     await resume();
   };
 
+  let scrollTimeout = null;
+  const handleScrollEvent = async () => {
+    if (scrollTimeout) return resume();
+    const prevScrollHeightResp = await client.send('Runtime.evaluate', {
+      expression: 'document.body.scrollHeight',
```

Comment:
This code assumes that `<body>` is the scrolling element which is not always the case. It's not always `<html>` neither. Might I suggest `document.scrollingElement` instead? See https://dev.opera.com/articles/fixing-the-scrolltop-bug/.

Reply:
The complexity of the web never cease to amaze me :) Thanks, I'll update to `document.scrollingElement`.